### ISSUE TITLE
messing with other mods and rockets

### ISSUE THREAD

**Issue by schlosrat** [untoldwind/KontrolSystem2#19]
Hi, I updated to the latest version of KontrolSystem today, and now Stage Info doesn't work. When I remove your mod then Stage Info is able to work again. This wasn't a problem with the previous version of your mod, and Stage Info hasn't changed (still at 0.7). Could you please take a look at what's different in KontrolSystem2 0.1.8 that might interfere with other mods?

**Comment by untoldwind** [comment]
I suppose you mean this on: https://spacedock.info/mod/3272/Stage%20Info
Just tested the 0.7 version and the 0.1.8 from spacedock on my machine and both seem to work quite well with each other. Can you pin-point which version of KontrolSystem is working? I don't really see anything of relevance in the changes from 0.1.7 -> 0.1.8 (that was mostly a bug-fix release)

Have you checked if there is something wired in the logs?
That is the `Ksp2.log` (the big one) an `BepInEx/LogOutput.log` (more focused on the mod-related stuff).

Somewhere at the end of the `BepInEx/LogOutput.log` should be something like:
```
[Info   :ToolbarBackend] Added appbar button: BTN-KontrolSystem
[Info   :ToolbarBackend] Added appbar button: BTN-StageInfoBtn
```

Deleting the `BepInEx/cache` folder might also help

**Comment by schlosrat** [comment]
I'm thinking this might be general bugginess in Stage Info. I just put 0.1.8 back into my plugins folder, launched the game and started a new campaign, and observed it all working normally. That said, just a few minutes ago I was seeing other weird problems with Stage Info without any version of KS installed. Whatever the issues are, I believe now they were only a coincidence that I saw them first appear when I updated KS.

Thanks for checking, and for the tips on looking in the logs for clues.
